src/config.json

```diff
@@ -19,7 +19,9 @@
         "extension_registry": "https://s3.amazonaws.com/extend.brackets/registry.json",
         "extension_url": "https://s3.amazonaws.com/extend.brackets/{0}/{0}-{1}.zip",
         "linting.enabled_by_default": true,
-        "build_timestamp": ""
+        "build_timestamp": "",
+        "healthDataServerURL" : "http://10.192.16.236:8080",
```

Comment:
This should be an HTTPS URL to ensure the user's data isn't sniffed over the wire. I know it's anonymous, aggregate, etc., but people still may not want that stuff visible to everyone on their subnet.

Also, can we get a real domain name instead of a bare IP address?  Bare IP looks kinda sketchy, and might limit our options for migrating servers if we need to later.